api_tests/collections/test_views.py

```diff
@@ -1822,6 +1822,16 @@ def test_post_mixed_nodes(self):
         assert_not_in(self.contributor_node._id, ids)
         assert_in(self.linked_node._id, ids)
 
+    def test_post_node_already_linked(self):
+        number_of_links = len(self.collection.nodes)
+        res = self.app.post_json_api(
+            self.url, self.payload([self.linked_node._id]),
+            auth=self.user.auth
+        )
+
+        assert_equal(res.status_code, 201)
```

Comment:
It looks like 204 might be more appropriate:

> 204 No Content
> ...
> Note: This is the appropriate response to a POST request sent to a URL from a to-many relationship link when that relationship already exists...

http://jsonapi.org/format/#crud-updating-relationship-responses-204